Assets/Scripts/CoinsUIController.cs

```diff
@@ -46,7 +46,10 @@ void Start()
             var activePlayers = NetworkManager.GetActivePlayerNumbers();
             for (int i = 0; i < GameConstants.MAX_ONLINE_PLAYERS_IN_GAME; ++i)
             {
-                mVoteUIElements[i].SetActive(activePlayers[i]);
+                if (!activePlayers[i])
+                {
+                    DisableUI(i);
+                }
             }
 
             StartCoroutine(HideInstructions());

```

Comment:
For now it's harmless and will probably be ignored so I wouldn't hold up this PR for that to get taken care of. If you think it hurts the final build we can strip it out in another PR